hazelcast-jet-core/src/main/java/com/hazelcast/jet/aggregate/AggregateOperationBuilder.java

```diff
@@ -54,12 +54,12 @@
     }
 
     /**
-     * Registers the supplied {@code accumulate} primitive. Also selects the
-     * fixed-arity variant of the aggregate operation.
+     * Registers the {@link AggregateOperation1#accumulateFn()} accumulate}
+     * primitive. Also selects the fixed-arity variant of the aggregate
+     * operation.
      * <p>
-     * This method is synonymous with {@link #andAccumulate0(
-     *BiConsumerEx)}, but makes more sense when defining a
-     * simple, arity-1 aggregate operation.
+     * This method is synonymous with {@link #andAccumulate0(BiConsumerEx)},
```

Comment:
In case you didn't consciously decide against it: maybe add labels to all such Javadoc links, to be consistent inside this entire class.

Reply:
Thanks, saved me another PR :)